app/views/availabilities/places/edit.html.erb

```diff
@@ -0,0 +1,72 @@
+<div class="container-fluid places-container">
```

Comment:
Put complex rendering logic in helpers.

Reply:
Do the same for the `index.html.erb` file

Reply:
which logic?

Reply:
@donjar bump

Reply:
Done